UVAtlasTool/Mesh.cpp

```diff
@@ -1087,6 +1089,59 @@ HRESULT Mesh::GetVertexBuffer(_Inout_ DirectX::VBWriter& writer) const
     return S_OK;
 }
 
+//--------------------------------------------------------------------------------------
+void Mesh::ExportToOBJ(const wchar_t* szFileName) const
+{
+    std::wstring ws(szFileName);
+    ExportToOBJ(std::string(ws.begin(), ws.end()));
+}
+/// Write to file
+void Mesh::ExportToOBJ(std::string filePath) const
+{
+    std::ofstream os;
+    os.open(filePath);
+    ExportToOBJ(os);
+}
+/// Writing to cout or to a file stream
+void Mesh::ExportToOBJ(std::ostream& os) const
+{
+    if (!mtlFileName.empty())
+        os << "mtllib ./" << std::string(mtlFileName.begin(), mtlFileName.end()) << ".mtl" << std::endl; // https://stackoverflow.com/questions/4804298/how-to-convert-wstring-into-string converting wstring to string this way works for almost everything except chinese characters
```

Comment:
Why not just use ``wofstream`` and ``wstring``?

Reply:
Personal preference. 
Made it wide. done